src/oauth2.h

```diff
@@ -26,29 +26,32 @@
 
 namespace google {
 
+constexpr const char kDefaultTokenEndpoint[] =
```

Comment:
Nice.

Reply:
Except we'd want to wrap it to an anonymous namespace — this isn't a symbol we want to export.

Reply:
Done.